mmv1/products/dns/api.yaml

```diff
@@ -37,6 +37,13 @@ objects:
       A zone is a subtree of the DNS namespace under one administrative
       responsibility. A ManagedZone is a resource that represents a DNS zone
       hosted by the Cloud DNS service.
+    iam_policy: !ruby/object:Api::Resource::IamPolicy
+      exclude: false
+      method_name_separator: ':'
+      fetch_iam_policy_verb: :POST
+      parent_resource_attribute: 'managed_zone'
```

Comment:
I'm not real familiar with the IAM-specific config here, but could you confirm that `managed_zone` is correct? I was expecting to see a field called `managed_zone` on this object, but didn't see one. The tests do appear to be passing, so I'm assuming everything works.

Reply:
Thanks for reviewing, referring to https://googlecloudplatform.github.io/magic-modules/docs/how-to/add-mmv1-iam/.

The parent resource is called ManagedZone, which -> managed_zone

Reply:
Ah ok, that makes sense, thanks for the link!